### PR TITLE
AST-2028 - Added sticky add to cart to astra theme

### PR DESCRIPTION
**Pull request by nathanrodrigues2111**
### Description
Remove sticky add to cart and moved to astra theme

Also includes :
-  https://brainstormforce.atlassian.net/browse/AST-2029
- https://brainstormforce.atlassian.net/browse/AST-2104

### Screenshots
- 

### Types of changes
New feature (non-breaking change which adds functionality)

### How has this been tested?
- Checked all options in backend.
- Checked if all options worked in frontend when applied 
- Checked if all code is removed from astra addons and moved to pro 

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: inc/compatibility/woocommerce/customizer/sections/layout/class-astra-woo-shop-single-layout-configs.php [hunk 1]
```diff
@@ -144,6 +153,235 @@ public function register_configuration( $configurations, $wp_customize ) {
...
+				/**
+				* Option: Sticky add to cart.
+				*/
+				array(
+					'name'        => ASTRA_THEME_SETTINGS . '[single-product-sticky-add-to-cart]',
+					'default'     => astra_get_option( 'single-product-sticky-add-to-cart' ),
+					'type'        => 'control',
+					'section'     => 'section-woo-shop-single',
+					'title'       => __( 'Enable Sticky Add to Cart', 'astra' ),
+					'description' => __( 'This works for only desktop devices', 'astra' ),
```
Comment (imnavanath) [inline inc/compatibility/woocommerce/customizer/sections/layout/class-astra-woo-shop-single-layout-configs.php:180]: Once responsive UI done we need to remove this description.

Comment (nathanrodrigues2111) [reply inc/compatibility/woocommerce/customizer/sections/layout/class-astra-woo-shop-single-layout-configs.php]: Removed @imnavanath

File: inc/compatibility/woocommerce/class-astra-woocommerce.php [hunk 2]
```diff
@@ -527,6 +538,13 @@ public function woo_filter_style( $styles ) {
 						'media'   => 'all',
 						'has_rtl' => true,
 					),
+					'sticky-add-to-cart'      => array(
+						'src'     => $css_uri . 'sticky-add-to-cart' . $file_prefix . '.css',
+						'deps'    => '',
+						'version' => ASTRA_THEME_VERSION,
+						'media'   => 'all',
+						'has_rtl' => true,
+					),
```
Comment (imnavanath) [inline inc/compatibility/woocommerce/class-astra-woocommerce.php:547]: We will load this CSS conditionally, if this option is enabled then only we will add.

File: inc/compatibility/woocommerce/class-astra-woocommerce.php [hunk 3]
```diff
@@ -503,6 +506,14 @@ public function woo_filter_style( $styles ) {
...
```
Comment (imnavanath) [inline inc/compatibility/woocommerce/class-astra-woocommerce.php:515]: We will load this CSS conditionally, if this option is enabled then only we will add.

File: inc/compatibility/woocommerce/customizer/sections/layout/class-astra-woo-shop-single-layout-configs.php [hunk 4]
```diff
@@ -193,6 +430,44 @@ public function register_configuration( $configurations, $wp_customize ) {
...
+					'help'     => __( 'Note: To get design settings make sure to enable sticky add to cart.', 'astra' ),
+					'context'  => array(
+						Astra_Builder_Helper::$design_tab_config,
+						array(
+							'setting'  => ASTRA_THEME_SETTINGS . '[single-product-sticky-add-to-cart]',
+							'operator' => '==',
+							'value'    => false,
+						),
+					),
+				);
```
Comment (imnavanath) [inline inc/compatibility/woocommerce/customizer/sections/layout/class-astra-woo-shop-single-layout-configs.php:451]: The reason of adding this notice is to not show blank "Design" tab in free theme, but as we are moving most of options to free theme, just make sure if this notice required or not. if there are other design option are coming directly in design tab then we can skip this. https://share.bsf.io/E0uZGEPe

Comment (imnavanath) [reply inc/compatibility/woocommerce/customizer/sections/layout/class-astra-woo-shop-single-layout-configs.php:451]: Required this, so resolving it...

File: inc/compatibility/woocommerce/class-astra-woocommerce.php (hunk 3 above)
Comment (imnavanath) [reply inc/compatibility/woocommerce/class-astra-woocommerce.php]: Done

**Review by imnavanath — APPROVED** [review]

**Review by imnavanath — APPROVED** [review]

**Pull request merged**